docker-compose.override.yml

```diff
@@ -76,26 +94,29 @@ services:
       exit 0;
       "
 
+  # Dev changes for the scheduler
+  scheduler:
+    <<: *airflow-common
+    depends_on:
+      - postgres
+      - s3
+    environment:
+      _AIRFLOW_WWW_USER_CREATE: "true"
+      _AIRFLOW_WWW_USER_USERNAME: airflow
+      _AIRFLOW_WWW_USER_PASSWORD: airflow
+      _AIRFLOW_WWW_USER_FIRSTNAME: Air
+      _AIRFLOW_WWW_USER_LASTNAME: Flow
+      _AIRFLOW_WWW_USER_EMAIL: airflow@example.com
+
   # Dev changes for the webserver container
   webserver:
+    <<: *airflow-common
     depends_on:
       - postgres
       - s3
-    build:
-      context: .
-      args:
-        - REQUIREMENTS_FILE=requirements_dev.txt
-        - PROJECT_PY_VERSION=${PROJECT_PY_VERSION}
-        - PROJECT_AIRFLOW_VERSION=${PROJECT_AIRFLOW_VERSION}
-      dockerfile: docker/airflow/Dockerfile
-    restart: on-failure
-    # This command ensures an admin account is created on startup
-    command: init
-    volumes:
-      - ./openverse_catalog:/usr/local/airflow/openverse_catalog
-    stdin_open: true
-    tty: true
+      - scheduler
 
 volumes:
   postgres:
   minio:
+  ipython:
```

Comment:
This is for retaining `ipython` history across docker restarts. Previously we were mounting a local folder to accomplish this (which I'm not even sure this actually worked as expected due to permissions issues) but now that we're running as the `airflow` user within the container it's a LOT more difficult to try and manage mounting that and making it write-accessible to the `airflow` user. It's way easier to simply add a volume mount here instead, since docker manages those entirely and permissions will be preserved. The downside there is that `just recreate` will destroy that history, but I still find it useful to retain the history where possible during development!